.changelog/11958.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+Fix a bug where a read replica node can trigger a raft election and become a leader..
```

Comment:
[nit]
```suggestion
```release-note:bug
Upgrade to raft `1.3.3` which fixes a bug where a read replica node can trigger a raft election and become a leader.
```